meta-resin-common/recipes-support/resin-device-register/resin-device-register/resin-device-register

```diff
@@ -44,25 +46,28 @@ fi
 while true; do
     if curl -sL -w '%{http_code}' $API_ENDPOINT/ping -o /dev/null | egrep "20[0-9]|30[0-9]" >/dev/null
     then
-        read api_key user_id application_id device_type uuid <<<$(jq -r '.apiKey,.userId,.applicationId,.deviceType // "raspberry-pi",.uuid' $CONFIG_PATH)
+        read api_key user_id application_id device_type registered_at <<<$(jq -r '.apiKey,.userId,.applicationId,.deviceType // "raspberry-pi",.registered_at // empty' $CONFIG_PATH)
+
+        if [ -n "${registered_at}" ]; then
+            echo "[INFO] resin-device-register : Device already registered."
+            exit 0
+        fi
 
         status_code=$(curl -s -X POST -w "%{http_code}" \
-                    -o /var/log/register-device-curl.log \
-                    -d "user=$user_id" \
-                    -d "application=$application_id" \
-                    -d "uuid=$uuid"  \
+                    --output $LOG_FILE \
+                    --data-urlencode "user=$user_id" \
+                    --data-urlencode "application=$application_id" \
                     --data-urlencode "device_type=$device_type" \
-                    "$API_ENDPOINT/ewa/device?apikey=$api_key")
+                    "$API_ENDPOINT/device/register?apikey=$api_key")
 
         if [ "$status_code" -eq 201 ]; then
-            device_id=$(jq -r '.id' /var/log/register-device-curl.log)
+            device_id=$(jq -r '.id' $LOG_FILE)
+            uuid=$(jq -r '.uuid' $LOG_FILE)
+            api_key=$(jq -r '.api_key' $LOG_FILE)
             config_json=`cat $CONFIG_PATH`
```

Comment:
I remember that we have a sqlite DB where UUID is stored along with some config. Does it make sense to use it and not split the data?
I see that it wasn't changed in this PR, so it's just a suggestion.